presto-main/src/test/java/com/facebook/presto/operator/GroupByHashYieldAssertion.java

```diff
@@ -56,9 +56,9 @@
 
     private GroupByHashYieldAssertion() {}
 
-    public static List<Page> createPagesWithDistinctHashKeys(Type type, int pageCount, int positionCountPerPage)
+    public static List<Page> createPages(Type type, int pageCount, int positionCountPerPage, boolean hashEnabled)
```

Comment:
It's not necessary to change the interface given whether we have hash enabled is irrelevant to the purpose of the test. Given `hashEnabled` is by default `true`, I think by passing the hash channel into the operator factory can make it work.